resources/chfs.rb

```diff
@@ -0,0 +1,19 @@
+
+actions :update, :resize
+default_action :resize
+
```

Comment:
Some of the mandatory attributes should be marked as such with `:required => true`.